extension/chrome/elements/compose.ts

```diff
@@ -217,6 +223,16 @@ export class ComposeView extends View {
   };
 
   public isFesUsed = () => Boolean(this.fesUrl);
+
+  private setInputLimit(el: HTMLElement, ev: JQuery.Event, limit: number) {
+    const clipboardEvent = ev.originalEvent as ClipboardEvent;
```

Comment:
can you try to address the element with `ev.currentTarget`, not `el`?
I have a suspicion that `el` is actually `ComposeView`, not the element pasted to.

Reply:
> can you try to address the element with ev.currentTarget, not el?

Sorry I didn't get it.

> I have a suspicion that `el` is actually ComposeView, not the element pasted to

When I `console.log()` the `el` variable, the console returns the DOM tree of the element, for example, the intro_input or input_text.

![image](https://user-images.githubusercontent.com/46025304/210224134-115a6a16-aa57-4262-a594-128041aecbed.png)